src/data/sql-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const sqlQuiz = [
+  {
+    Question: "In SQL, which of the following is not a commonly used SQL constraints",  
+    Answer: "UNIQUE",
+    Distractor1: "CHECK",
+    Distractor2: "DEFAULT",
+    Distractor3: "UPDATE",
```

Comment:
this is supposed to be the correct answer though. remember that distractors are for wrong answers only

```suggestion
    Distractor3: "UNIQUE",
```